components/threading/global_threads/thread_item/thread_item.scss

```diff
@@ -108,7 +114,7 @@
         font-size: 14px;
         font-weight: 600;
         gap: 10px;
-        grid-template-columns: minmax(0, min-content) 1fr min-content;
+        grid-template-columns: minmax(0, min-content) minmax(0, 1fr) min-content;
```

Comment:
It doesn't seem to me that this would solve the issue, I would expect that the name should be truncated not the tag shrink.

![Screenshot 2023-02-03 at 2 25 13 PM](https://user-images.githubusercontent.com/3829551/216603332-406563b4-6c18-49a9-8887-e8f829d58c86.png)

Reply:
@michelengelen can you also add a space between the urgent badge and the channel tag,
it seems that that one also got affected by the new tags component.

Reply:
OK, will check that

Reply:
@koox00 I fixed both issues .. could you re-check please?

Reply:
BTW, the `fit-content: 40%` will force it to take 40% even when it could take more.